### ISSUE TITLE
How to use a fixed learning rate?

### ISSUE THREAD

**Issue by tserdar** [ultralytics/yolov5#3824]
## ❔Question

Is setting `lrf = 1 (OneCycleLR learning rate (lr0 * lrf)` in the hyperparameters the correct way?

**Comment by glenn-jocher** [comment]
@rocketsfallonrocketfalls if you're asking how to never modify the learning rate you can just comment out the LR scheduler in train.py.
